db/migrate/20230329183243_add_u_sers_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUSersRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```

Comment:
- Hi :clap: , in my previous review, the same solution is applicable here. this should be 
`add_reference :likes, :user, null: false, foreign_key: true` and not 
`add_reference :likes, :users, null: false, foreign_key: true`. let's fix this please :pray: